src/main/java/org/truffleruby/core/TruffleSystemNodes.java

```diff
@@ -95,31 +93,6 @@ protected RubyArray envVars() {
 
     }
 
-    @Primitive(name = "java_get_env")
```

Comment:
This we actually want to keep both as a debugging aid and potentially to implement `ENV#[]` with no native access.
It should probably be a core method though, like `get_java_property` below.
I'll add it as a core method.